lms/templates/api/canvas/page.html.jinja2

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang=en>
+  <head>
+    <meta charset=utf-8>
+    <title>{{title}}</title>
+  </head>
+  <body>
+    <h1>{{title}}</h1>
```

Comment:
This doesn't improve the actual output that much but later changes would be more obvious with the template in place.